### ISSUE TITLE
Edit mode breaks if I rejoin the world.

### ISSUE THREAD

**Issue by AngelOfCreation** [McJty/LostCities#617]
I got edit mode to work properly... sort of. It breaks if I exit the game and rejoin. I have to create a new world every time I want to edit a part because if I leave the world and rejoin, it no longer registers the world as having any parts. locatepart doesn't return any results, editpart does nothing, createpart says that no parts exist to create, etc.

Playing with the latest version of DeceasedCraft, 1.18.2
https://www.curseforge.com/minecraft/modpacks/deceasedcraft

**Comment by McJty** [comment]
Yes, this is a limitation that's kinda hard to fix. Not sure what to do about this for now

**Comment by AngelOfCreation** [comment]
I don't understand, wouldn't edit mode just be a boolean tag in a saves level.dat file that enables the commands? I tried searching through my save with NBTStudio but I couldn't find it.

**Comment by McJty** [comment]
No, to be able to do all the commands the mod records information when buildings are created (i.e. when chunks are generated). So edit mode can only work for chunks that have been generated in that run. If you would move away in the same world to new chunks you would be able to continue edit mode there. But not in chunks that were already there because I don't keep that information when the game restarts
